openlibrary/plugins/openlibrary/processors.py

```diff
@@ -5,7 +5,10 @@
 import web
 
 from openlibrary.accounts import get_current_user
+from openlibrary.core import cache
 from openlibrary.core.processors import ReadableUrlProcessor
+from openlibrary.plugins.openlibrary.home import caching_prethread
```

Comment:
**Reviewer:** Is it okay to use this `caching_prethread`?

I more or less re-implemented the homepage caching in this new processor, which uses `caching_prethread`.